### DISCUSSION TITLE
Problem

### DISCUSSION THREAD

**Discussion by Ziyad197** [community/community#52073] [GitHub Education]
What is solution?
![AB47DFB8-55A6-4E65-B14A-A238E96FE906](https://user-images.githubusercontent.com/128281255/230113984-a30a5590-44da-45c6-85e4-ca2a11e0ab6a.png)
![099FF5E7-5AB5-4E74-AEFE-471AC3F1C2C3](https://user-images.githubusercontent.com/128281255/230114025-2e58618d-09c7-4e06-b2eb-e7a4ee30a4bf.png)

**Comment by notahumans-official** [comment]
Ask your teacher, maybe they deleted your assignment.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @ZiyadAlamri , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!

**Comment by samus-aran** [comment]
Hi @ZiyadAlamri 👋  Did you reach out to your teacher to resolve this?
